package/prerm

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+install_dir=/usr/local/nebula
+daemons=(storaged metad graphd)
+for daemon in ${daemons[@]}
+do
+    if [[ ! -f $install_dir/scripts/nebula.service ]] ; then
+        $install_dir/scripts/nebula.service stop ${daemon}
```

Comment:
This operation can be dangerous in a production environment. I suggest that we do not do this implicitly.